README.md

```diff
@@ -229,6 +229,22 @@ peerConnection.addEventListener('track', event => {
 signalingClient.open();
 ```
 
+##### Add adapter.js to increase compatibility between browsers
+
+In your html, add the following:
+
+```
+<head>
+  ...
+  <script src="https://webrtc.github.io/adapter/adapter-latest.js"></script>
+  ...
+</head>
+```
+
+> adapter.js is a shim to insulate apps from spec changes and prefix differences in WebRTC. The prefix differences are mostly gone these days but differences in behaviour between browsers remain.
+
+More details: https://github.com/webrtcHacks/adapter
```

Comment:
I think it is generally recommended to give links a more human readable name. e.g. `[adaptor.js docs](https://github.com/webrtcHacks/adapter)`